src/app/pages/vendor-dashboard/components/vendor-filter/vendor-filter.component.html

```diff
@@ -1 +1,34 @@
-<p>vendor-filter works!</p>
+<body style="background-color:rgb(227, 227, 227);">
+    <div>
+    <h3>Status</h3>
```

Comment:
Can we have a screen shot of what the view looks like?

Reply:
let me add the CSS then i will post one